editor/find_in_files.cpp

```diff
@@ -377,6 +415,26 @@ FindInFilesDialog::FindInFilesDialog() {
 		gc->add_child(hbc);
 	}
 
+	Label *includes_label = memnew(Label);
+	includes_label->set_text(TTR("Includes:"));
+	includes_label->set_tooltip_text(TTR("Include the files with the following expressions. Use \",\" to separate."));
+	gc->add_child(includes_label);
+
+	_includes_line_edit = memnew(LineEdit);
+	_includes_line_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	_includes_line_edit->set_placeholder("example: scripts,scenes/*/test.gd");
+	gc->add_child(_includes_line_edit);
+
+	Label *excludes_label = memnew(Label);
+	excludes_label->set_text(TTR("Excludes:"));
+	excludes_label->set_tooltip_text(TTR("Exclude the files with the following expressions. Use \",\" to separate."));
+	gc->add_child(excludes_label);
+
+	_excludes_line_edit = memnew(LineEdit);
+	_excludes_line_edit->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	_excludes_line_edit->set_placeholder("example: addons,scenes/test/*.gd");
```

Comment:
To only exclude the root addons folder, the pattern for it should be `//addons/` instead of `addons`.